### PR TITLE
Split spec common actions support helper

### PR DESCRIPTION
**Pull request by aamarill**
This PR is for issue #2644 posted by @voodoorai2000.

Notes
=====================
> Please review the way I currently include the modules into the "CommonActions" module. I feel like there should be a cleaner way to include them. 

> Please review the current location of the relocated methods. On top of each method there is a comment with which files use that method. I hope that helps in figuring out where they should go.

Thanks! 🙌

### CODE REVIEW

File: spec/support/common_actions/budgets.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Budgets
+  # spec/features/budgets/ballots_spec.rb
+  def expect_message_organizations_cannot_vote
+    #expect(page).to have_content 'Organisations are not permitted to vote.'
```
Comment (aitbw) [inline spec/support/common_actions/budgets.rb]: If this expectation is not being used, I think it'd be better to remove it altogether

File: spec/support/common_actions/users.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    click_link 'Sign in'
+
+    fill_in 'user_login', with: user.email
+    fill_in 'user_password', with: user.password
+
+    click_button 'Enter'
+    visit new_officing_residence_path
+  end
+
+
```
Comment (aitbw) [inline spec/support/common_actions/users.rb]: Please, remove this trailing line

**Comment by voodoorai2000** [comment]
This is magnificent!
Thank you so much @aamarill 👏 

The way to include the modules is great 👍 
And the locations too 👌😌

Thank for the comments with the references to the code, they helped 😌
Feel free to clean them up
With that and @aitbw's comments I think we are ready to merge 👍

**Comment by aamarill** [comment]
Sorry for the delay! Let me know if I missed anything 😄 
Hope everyone has a great week!

**Comment by voodoorai2000** [comment]
Awesome! Thank @aamarill 🎉

**Comment by aamarill** [comment]
It was truly a pleasure to work with you all 😄 🙏 Thank you for your words of encouragement and support!

**Pull request merged**